lerobot/scripts/visualize_dataset.py

```diff
@@ -154,8 +166,20 @@ def visualize_dataset(
 
             # display each camera image
             for key in dataset.camera_keys:
-                # TODO(rcadene): add `.compress()`? is it lossless?
-                rr.log(key, rr.Image(to_hwc_uint8_numpy(batch[key][i])))
+                if isinstance(batch[key], torch.Tensor):
+                    rr.log(key, rr.Image(to_hwc_uint8_numpy(batch[key][i])))
+                elif "path" in batch[key] and "timestamp" in batch[key]:
+                    if sent_videos.get(key) != batch[key]["path"][i]:
+                        sent_videos[key] = batch[key]["path"][i]
+                        rr.log(key, rr.AssetVideo(path=dataset.videos_dir.parent / batch[key]["path"][i]))
```

Comment:
This should be simpler to do with the version 2 of our dataset (#461) but let's keep it *as is* here and I'll refactor when it gets merged.